cpp/src/io/json/nested_json.hpp

```diff
@@ -23,8 +23,11 @@
 #include <cudf/utilities/error.hpp>
 #include <cudf/utilities/export.hpp>
 
+#include <rmm/exec_policy.hpp>
 #include <rmm/resource_ref.hpp>
 
+#include <thrust/sequence.h>
+
```

Comment:
Move these 2 includes to source file. it is not required here.

Reply:
Addressed.